frontend/src/app/modules/header/header-item/header-item.component.sass

```diff
@@ -33,12 +33,25 @@ header
 .tab-panel
     height: 100%
 
+.link-btn
+    justify-content: center
+    height: 35px
+
+::ng-deep .mat-tab-nav-bar .mat-tab-links .mat-tab-link
+    padding: 0 15px 0 15px
```

Comment:
are you sure that you didn't break anything?

Reply:
yep. I checked all another pages - any broken murkups

Reply:
anyway wrap it with an additional class